modules/ding_frontpage/ding_frontpage.pages_default.inc

```diff
@@ -44,7 +44,7 @@ function ding_frontpage_default_page_manager_pages() {
     'relationships' => array(),
   );
   $display = new panels_display();
-  $display->layout = 'ding_layout_1';
```

Comment:
Did you branch out from `ddbcms` and not `development`?

Reply:
Yes, this was branched from development.